includes/specials/SpecialUserlogin.php

```diff
@@ -738,8 +738,8 @@ public function authenticateUserData() {
 			$retval = self::SUCCESS;
 		}
 
-		if ( in_array( $retval, [ self::SUCCESS, self::RESET_PASS ] ) ) {
-			wfRunHooks( 'LoginSuccessModifyRetval', [ $u->getName(), $this->mPassword, &$retval ] );
+		if ( !in_array( $retval, [ self::SUCCESS, self::RESET_PASS ] ) ) {
```

Comment:
I repurposed this hook because it seems like it was a noop since its original intention was overwritten above with 
```php
} elseif ( $wgEnableHeliosExt && Wikia\Helios\User::wasResetPassAuth( $this->mUsername, $this->mPassword ) ) {
```

to set the correct status of the function. @garthwebb do you remember if there was there a reason this hook invocation was left in place after adding the above `elseif`? It was added in 2da99740a249d8dbe89be6626a30c248f8b19ee4